content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -32,6 +32,12 @@ in that there is no corresponding InfluxDB OSS release.
 - Ensure a newline is printed after a successful copy during [restoration](/enterprise_influxdb/v1.9/administration/backup-and-restore/).
 - Make `entropy show` expiration times consistent with `show-shards`.
 
+### Other changes
+
+- Remove `influx_stress` tool. (This had been marked as deprecated since version 1.2.)
```

Comment:
Does "Deprecated since version 1.2" work...or was this tool "marked" to deprecate and not actually deprecated?

Reply:
The former should work. 👍